docs/usage/registry-cache/configuration.md

```diff
@@ -92,6 +92,15 @@ The `providerConfig.caches[].garbageCollection.ttl` field is the time to live of
 
 The `providerConfig.caches[].secretReferenceName` is the name of the reference for the Secret containing the upstream registry credentials. To cache images from a private registry, credentials to the upstream registry should be supplied. For more details, see [How to provide credentials for upstream registry](upstream-credentials.md#how-to-provide-credentials-for-upstream-registry).
 
+The `providerConfig.caches[].proxy.httpProxy` field contains settings for a proxy which is used by the registry cache.
+The registry-cache extension sets this value as `HTTP_PROXY` ENV in the StatefulSet.
```

Comment:
IMO this is an implementation detail and shouldn't be part of the external docs. Today it is env var of a StatefulSet, tomorrow it can be something else on a Deployment for example.
At the end, end users should be interested on what the field does but not how it is implemented under the hood.

```suggestion
```

Reply:
fixed